### PR TITLE
PIM-4499 : mass upload

### PR DESCRIPTION
**Pull request by jmleroux**
| Q                  | A
| ------------------ | ---
| Migration script?  |n
| Behats             |n
| Specs & Static     |y
| Changelog updated  |n

### CODE REVIEW

**Comment by phaseinducer** [comment]
GTM

File: src/Pim/Bundle/UIBundle/Resources/public/lib/dropzonejs/dist/basic.css [hunk 1]
```diff
@@ -0,0 +1,39 @@
+/*
+ * The MIT License
+ * Copyright (c) 2012 Matias Meno <m@tias.me>
+ */
+.dropzone, .dropzone * {
+  box-sizing: border-box; }
+
+.dropzone {
```
Comment (sumbobyboys) [inline src/Pim/Bundle/UIBundle/Resources/public/lib/dropzonejs/dist/basic.css]: wow, you are not responsible, but this is pretty ugly

File: src/Pim/Bundle/UIBundle/Resources/translations/jsmessages.en.yml [hunk 2]
```diff
@@ -36,6 +36,11 @@ Localized value: Localized value
 Columns: Columns
 Datagrid Configuration: Datagrid Configuration
 Apply: Apply
+error: Error
```
Comment (sumbobyboys) [inline src/Pim/Bundle/UIBundle/Resources/translations/jsmessages.en.yml]: please use proper translation keys

File: src/Pim/Bundle/UIBundle/Resources/translations/messages.en.yml [hunk 3]
```diff
@@ -18,11 +18,21 @@ Channel list:         Channel list
 Select file:          Select file
 Change file:          Change file
 Remove file:          Remove file
+Add files:            Add files
+
+View: View
```
Comment (sumbobyboys) [inline src/Pim/Bundle/UIBundle/Resources/translations/messages.en.yml]: same here

**Comment by grena** [comment]
GTM

**Pull request merged**